package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "govuk_design_system",
+  "version": "0.1.0",
```

Comment:
Do we need to define this in order to specify dependencies? I don't think it's useful (at least at the minute) – we're not going to use it or remember to update it?

Reply:
It's not mandatory and I don't this it's useful at this point.